src/main/java/com/lambdaschool/oktafoundation/repository/MemberReactionRepository.java

```diff
@@ -32,4 +33,13 @@
      */
     List<MemberReactions> getMemberReactionsByMember(Member m);
 
+    // Get initial list of all memberReactions within date range
+    // init count object for all members which is a map from memberID to a list of value that the member gave out.
+
+//    Query to populate the alert lists, all memberreactions where reactionresolved is not true and activityid=7 (checkin)
+    @Query(
+            value = "SELECT * FROM MEMBERREACTIONS where reactionresolved=false AND activityid=7",
```

Comment:
Thanks for this change. we are going to have to come up with a more robust way of grabbing these checkin and checkout activities

Reply:
I would likely say mapping checkin/checkout by activityid is reliable- but for specific checkin/checkout for activities the same usage may not be as consistent